src/test/test.ts

```diff
@@ -0,0 +1,34 @@
+export const defaultAnsibleConfigurations = {
+  'ansible.useFullyQualifiedCollectionNames': true,
+  'ansibleLint.arguments': '',
+  'ansibleLint.enabled': false,
+  'ansibleLint.path': 'ansible-lint',
+  'ansibleNavigator.path': 'ansible-navigator',
+  'executionEnvironment.containerEngine': 'auto',
+  'executionEnvironment.enabled': false,
+
+  'executionEnvironment.image':
+    'quay.io/ansible/ansible-devtools-demo-ee:v0.1.0',
+
+  'executionEnvironment.pullPolicy': 'missing',
+  'python.activationScript': '',
+  'python.interpreterPath': 'python3',
+  'ansible.path': 'ansible',
+};
+
+// defaultAnsibleConfigurations.forEach((config) => {
```

Comment:
Maybe don't commit this?

Reply:
Same as above. The files are unnecessary. I will remove them.